src/Tutor.Web/Mappings/Enrollments/GroupProfile.cs

```diff
@@ -15,7 +15,10 @@ public GroupProfile()
         CreateMap<KnowledgeComponentMastery, KcmProgressDto>()
             .ForMember(dest => dest.DurationOfAllSessionsInMinutes, opt => opt.MapFrom(src =>
                 (src.SessionTracker.DurationOfAllSessions.Hours * 60) +
-                src.SessionTracker.DurationOfAllSessions.Minutes));
+                src.SessionTracker.DurationOfAllSessions.Minutes))
+            .ForMember(dest => dest.DurationOfAllPausesInMinutes, opt => opt.MapFrom(src =>
```

Comment:
Not sure if this is the correct place for this logic or if it is needed.

Does this give accurate results https://learn.microsoft.com/en-us/dotnet/api/system.timespan.totalminutes?view=net-7.0 ?